src/target/lpc546xx.c

```diff
@@ -98,55 +140,46 @@ bool lpc546xx_probe(target_s *t)
 {
 	const uint32_t chipid = target_mem_read32(t, LPC546XX_CHIPID);
 	uint32_t flash_size = 0;
+	uint32_t sram123_size = 0;
+
+#ifdef USE_LUT
+	const struct lpc546xx_device *device = lpc546xx_get_device(chipid);
+	if (!device)
+		return false;
 
+	flash_size = device->flash_kbytes * 1024U;
+	t->driver = device->designator;
+	/* All parts have 64kB SRAM0 (and 32kB SRAMX)
```

Comment:
```c
/*
 * Comment
 * Lines
 * Here
 */
```
Is how multi-line comments should be formatted, please move the opening line of the comment appropriately

Reply:
Fixed two multiline comments.